test/web-platform-tests/to-upstream/html/semantics/forms/the-input-element/checkbox-click-events.html

```diff
@@ -0,0 +1,74 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>Checkbox click events</title>
+<link rel="author" title="jeffcarp" href="mailto:gcarpenterv@gmail.com">
```

Comment:
I saw that some of the tests have author tags. Wasn't sure whether to fill them out or not.